R/chat.R

```diff
@@ -49,50 +55,79 @@ NULL
 #'   Briefly explain your work.
 #' ")
 new_chat_openai <- function(system_prompt = NULL,
+                            messages = NULL,
                             base_url = "https://api.openai.com/v1",
                             api_key = openai_key(),
-                            model = "gpt-4o-mini",
+                            model = NA,
```

Comment:
We'd normally make this `NULL` so you can later do `model <- model %||% "gpt-4o-mini"`

Reply:
Yeah, I was wondering if in this case it was worth preserving the ability to NOT have a `model` argument on the API call. We don't need this today but just in case there is an OpenAI-compatible-ish API in the future that gets popular...

Might be overthinking it...

I guess `NULL` can mean "choose for me" today, and if needed later we could have `NA` mean "explicitly don't include this parameter"? How does that strike you?

Reply:
I made it `NULL`